Link to the SIG interop tools terminology docs (#26)
cdfoundation/best-practices-site: content/en/learn/tools-terminology/_index.md

---
title: "Terminology Used by CI/CD Tools"
linkTitle: "Tools Terminology"
weight: 200
description: >
  By the SIG Interoperability - a catalog of tools terminology
---

# Introduction

> There are many ways to greet someone and as humans if we do not understand the word being used we have the ability to observe body language, process tone, and even touch. These many different natural inputs allow us as humans to establish shared vocabulary upon which we have been able to build successful components relevant to our way of living and social norms of interacting.

Reviewer: ```suggestion
There are many ways to greet someone. If we do not understand the words being used, we have the ability to observe body language, process tone, and even touch. These many different natural inputs allow us as humans to establish shared vocabulary relevant to our way of living and social norms of interacting.
```

Author: Thanks for the review @justinabrahms - this text is copy-pasted from the interoperability page as a way of introduction - https://github.com/cdfoundation/sig-interoperability/blob/main/docs/tools-terminology.md#terminology-used-by-cicd-tools-and-technologies. I'm happy to update it as you suggested, but shall we update it back in the sig interop page as well?

Reviewer: Sounds wonderful. :)

> Unfortunately for machines, this process is not so easy as we humans have to decide if we want to establish norms which we often surface when talking about machine interactions as protocols and best practices or requirements.

Reviewer: ```suggestion
Unfortunately for machines, this process is not so easy. We humans have to decide if we want to establish norms which surface when talking about machine interactions as protocols and best practices or requirements.
```

> Continuous Integration (CI) and Continuous Delivery (CD) practitioners have many tools at their disposal but it is often the case that what we call a pipeline in today’s tool of choice is not called the same thing in the tool we use tomorrow. Again, we can within our sphere of influence and interaction adjust for these nuances but machines talking to one another do not have that same luxury necessarily.

Reviewer: ```suggestion
Continuous Integration (CI) and Continuous Delivery (CD) practitioners have many tools at their disposal but it is often the case that what we call a pipeline in today’s tool of choice is not called the same thing in the tool we use tomorrow. Again, we can, within our sphere of influence and interaction, adjust for these nuances, but machines talking to one another do not have that same luxury.
```

> The CDF Interoperability Special Interest Group created a document to collect the basic terms used by CI/CD tools and technologies in order to work on establishing a shared vocabulary for us humans to communicate and collaborate better.

Reviewer: ```suggestion
The CDF [Interoperability Special Interest Group](https://github.com/cdfoundation/sig-interoperability/#readme) created a document to collect the basic terms used by CI/CD tools and technologies in order to work on establishing a shared vocabulary for us humans to communicate and collaborate better.
```
